content/shared/influxdb3-cli/create/plugin.md

```diff
@@ -39,4 +39,27 @@ You can use the following environment variables to set command options:
 | `INFLUXDB3_DATABASE_NAME` | `--database` |
 | `INFLUXDB3_AUTH_TOKEN`    | `--token`    |
 
-<!-- TODO: GET EXAMPLES -->
+## Examples
```

Comment:
`plugin` operations don't exist anymore. We should delete this and any other pages for `plugin` CRUD commands.